src/components/NewTodo/NewTodo.tsx

```diff
@@ -0,0 +1,63 @@
+import React, { useState } from 'react';
+import { SelectOption } from '../SelectOption/SelectOption';
+import { TextField } from '../TextField/TextField';
+
+interface Props {
+  onChangeTodo: (title: string, userId: number, completed?: boolean) => void;
+}
+
+export const NewTodo: React.FC<Props> = ({ onChangeTodo }) => {
+  const [count, setCount] = useState(0);
+  const [title, setTitle] = useState<string>('');
+  const [selectedUser, setSelectedUser] = useState<number>(0);
+  const [submited, setSubmited] = useState<boolean>(false);
+
+  const hasErrorTextField = title.trim() === '' && submited;
+  const hasErrorSelectOption = selectedUser === 0 && submited;
```

Comment:
1) 0 is a `falsy` value. So `0 === false` and `!false === true` in ur conditions
2) `''` - is not `falsy`, so u need to check the `length`
```suggestion
  const hasErrorTextField = !title.trim().length && submmited;
  const hasErrorSelectOption = !selectedUser && submmited;
```